manager/manager.go

```diff
@@ -134,6 +144,7 @@ func NewManager(opts Options) (*Manager, error) {
 
 	m := &Manager{
 		client:         opts.Client,
```

Comment:
This `client` is not used anymore right? Because if `createClient` is not set in opts, it is being set to return `opts.Client`

Reply:
It's still used for deleting CRs: https://github.com/cert-manager/csi-lib/pull/9/files#diff-f679b61276b44dc3f4712950536707c0389ee4c536fc29837c0202c6e81c1bbfR358

Reply:
```go
		if err := m.client.CertmanagerV1().CertificateRequests(toDelete.Namespace).Delete(ctx, toDelete.Name, metav1.DeleteOptions{}); err != nil {
```